test/test_torch.py

```diff
@@ -16448,16 +16447,160 @@ def test_svd_tall_some_col_maj(self, device, dtype):
     def test_svd_tall_all_col_maj(self, device, dtype):
         self._test_svd_helper((5, 20), False, True, device, dtype)
 
+
+# Below are fixtures and functions for generating tests related to unary
+# "universal functions (ufuncs)", a class of elementwise unary operations
+# This class can be further divided into "normal" and "floating" unary
+# ufuncs. The former does not participate in type promotion, while the latter
+# promotes integral types, including bool, to the default scalar type.
+
+# Helpers to organize ufunc metadata and improve test readability
+_ufunc_meta = collections.namedtuple('_ufunc_meta',
+                                     'is_floating alt_impl complex_on complex_out')
+def ufunc_meta(is_floating=False, alt_impl=None, complex_on=('cpu', 'cuda'), complex_out=('cpu', 'cuda')):
+    return _ufunc_meta(is_floating, alt_impl, complex_on, complex_out)
```

Comment:
Let's remove alt_impl for now, since it won't be used by this PR. (The future PR I included it for can add it back.)

Reply:
Sure! Sounds good.

Reply:
This is solved in the recent commit: https://github.com/pytorch/pytorch/pull/33322/commits/29ae207f362efe4040b598355cfe1db1fadb82ad. Thanks!